db/migrate/20181108225722_create_teams.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTeams < ActiveRecord::Migration[5.2]
+  def change
+    create_table :teams do |t|
+      t.string :nba_id
+      t.string :city
```

Comment:
I probably could have taken it out with this branch considering we're going leaner and can add that later. I think it will be nice to have city as a separate field to have more options for formatting as opposed to having one field that contains "Chicago Bulls".